masonry/src/widget/prose.rs

```diff
@@ -285,7 +285,7 @@ impl Widget for Prose {
     }
 
     fn accessibility_role(&self) -> Role {
-        Role::StaticText
+        Role::Label
```

Comment:
Is this the right role for `Prose`? I think `Paragraph` might be better (still not completely correct, of course, but closer to what the Prose actually represents).

Reply:
I wondered the same thing and found it deeply unsatisfying to use `Role::Label` there. But I don't think there's a single right answer and it will depend on what the prose is for ... there are several roles that could apply and I think it'll need to be configurable.

Reply:
I don't think `Label` can ever meaningfully apply to `Prose`, but I also don't know what the better role would be. The thing to test is what role does the browser use for something like the main article text of a body (e.g. the body of this comment

which has multiple paragraphs but is logically one node).

Reply:
Of course, the web splits each paragraph into its own node/widget. Maybe we need to be doing that?
I think for now, we need to choose one of paragraph or label. I'd lean *slightly* towards paragraph, but it's not a well-reasoned position.